src/Function.cpp

```diff
@@ -562,6 +570,43 @@ Function::OutputForwardDecl(std::ostream &out)
 	OutputHeader(out);
 	if(func_attr_inline)
 		out << " __attribute__((always_inline))";
+	if(func_attr_aligned){
+		int probability__BIGGEST_ALIGNMENT__ = 38;
+		bool use__BIGGEST_ALIGNMENT__ = rnd_flipcoin(probability__BIGGEST_ALIGNMENT__);
+		if (use__BIGGEST_ALIGNMENT__)
+			out << " __attribute__((aligned(__BIGGEST_ALIGNMENT__)))";
```

Comment:
I would just set `value = __BIGGEST_ALIGNMENT__` and do `out <<` later just once.

Reply:
Done